maquetado/index.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet" type="text/css" href="css/application.css">
+</head>
+
+<body>
+  <article class="book-detail">
+    <h3 class="book-title">Título del libro</h3> <span class="book-genre">(género)</span>
+    <hr class="separator">
```

Comment:
Remember to use as less html code as possible for clarity and reusability. In this case, try just to use just the h3 element and apply the separator as a decorator using only CSS (hint: you can use the ::before selector), that way you will have a single reusable html tag (the h3) that contains the text, and manage the decorative part of the text via CSS only. If you have any doubt about this ask me.